src/Components/Blazor/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -201,6 +205,14 @@
       <_DotNetHostFileName Condition=" '$(OS)' == 'Windows_NT' ">dotnet.exe</_DotNetHostFileName>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <_TypeGranularityLinkingConfig>$(BlazorIntermediateOutputPath)linker.typegranularityconfig.xml</_TypeGranularityLinkingConfig>
+    </PropertyGroup>
+    <GenerateTypeGranularityLinkingConfig Assemblies="@(_BlazorAssemblyToLink)" OutputPath="$(_TypeGranularityLinkingConfig)" />
```

Comment:
Invoking this target should be incremental. For most users, the contents of this file would almost never change. If you'd like to do less MSBuild, you can check this in and make an issue to make it so.

Reply:
This runs if and only if we're going to run the linker. Since the linker is orders of magnitude more expensive than this anyway, I didn't think it would achieve anything to make this bit incremental.

If you still think it's not right, please either file an issue so we track changing it in the future, or just let me know and I will do so. Thanks!